components/camel-pulsar/src/test/java/org/apache/camel/component/pulsar/PulsarComponentTest.java

```diff
@@ -7,7 +7,7 @@
 
     @Test
```

Comment:
We should properly delete this file

Reply:
Or actually write some tests. Which is what I'll do.